pytrec_eval/qrels.py

```diff
@@ -88,8 +88,12 @@ def _parseFile(self, otherQRelsPath):
         for line in fQRels:
             line = line.strip()
             if line == '': continue
-            split = line.split('\t')
-            topicId, docId, relevanceScore = split[0], split[2], float(split[3])
+            split = line.split()
+            try:
+              relevanceScore = float(split[3].strip('"'))
```

Comment:
could you please extract a private def `get_relevance_score`, so that the code gets a bit cleaner?
I know, i should rewrite everything and use a linter but I don't work on this topic anymore…

Reply:
I've made the change.